src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -5,74 +5,30 @@
+using System.Net.Http;
 using System.Security;
-using Newtonsoft.Json.Linq;
-using NJsonSchema;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Json.Schema;
 
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
     /// This class implements Test-Json command.
     /// </summary>
-    [Cmdlet(VerbsDiagnostic.Test, "Json", DefaultParameterSetName = JsonStringParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096609")]
+    [Cmdlet(VerbsDiagnostic.Test, "Json", DefaultParameterSetName = ParameterAttribute.AllParameterSets, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096609")]
     [OutputType(typeof(bool))]
     public class TestJsonCommand : PSCmdlet
     {
-        #region Parameter Set Names
-
-        private const string JsonStringParameterSet = "JsonString";
-        private const string JsonStringWithSchemaStringParameterSet = "JsonStringWithSchemaString";
-        private const string JsonStringWithSchemaFileParameterSet = "JsonStringWithSchemaFile";
-        private const string JsonPathParameterSet = "JsonPath";
-        private const string JsonPathWithSchemaStringParameterSet = "JsonPathWithSchemaString";
-        private const string JsonPathWithSchemaFileParameterSet = "JsonPathWithSchemaFile";
-        private const string JsonLiteralPathParameterSet = "JsonLiteralPath";
-        private const string JsonLiteralPathWithSchemaStringParameterSet = "JsonLiteralPathWithSchemaString";
-        private const string JsonLiteralPathWithSchemaFileParameterSet = "JsonLiteralPathWithSchemaFile";
-
-        #endregion
-
-        #region Parameters
+        private const string SchemaFileParameterSet = "SchemaFile";
+        private const string SchemaStringParameterSet = "SchemaString";
 
         /// <summary>
         /// Gets or sets JSON string to be validated.
         /// </summary>
-        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ParameterSetName = JsonStringParameterSet)]
-        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ParameterSetName = JsonStringWithSchemaStringParameterSet)]
-        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ParameterSetName = JsonStringWithSchemaFileParameterSet)]
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true)]
         public string Json { get; set; }
 
-        /// <summary>
-        /// Gets or sets JSON file path to be validated.
-        /// </summary>
-        [Parameter(Position = 0, Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = JsonPathParameterSet)]
-        [Parameter(Position = 0, Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = JsonPathWithSchemaStringParameterSet)]
-        [Parameter(Position = 0, Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = JsonPathWithSchemaFileParameterSet)]
-        public string Path { get; set; }
```

Comment:
Why do we remove this?

Reply:
I'm not sure.  This doesn't look like something I would have touched.  Maybe it was added separately after I made changes?

Reply:
yeah, looks like all of this was added separately in https://github.com/PowerShell/PowerShell/pull/19042

Reply:
Yes, it was wrong rebase here.

Reply:
I really don't like the idea of "JsonPath" being use here.  It causes confusion with JSON Path, the query syntax.

Reply:
It is another context. I don't think anyone will be misled, but you can suggest a better name if you like (in another PR).

Reply:
I'm filling all this back in (including the PS tests) and running locally to make sure, but I think I'm going to have to make updates to account for these new parameters and the new tests running.

Nice catch @iSazonov.

Reply:
Pushed.

Reply:
On the `JsonPath` thing, I realized after working with this for a bit that the parameter sets aren't exposed to the user.  I think we're okay, but it might be something to update in the future so it's clear to future devs.